docusaurus/docs/welcome/conventions.md

```diff
@@ -70,17 +70,17 @@ The folder structure convention favours co-location of assets and their requeste
                 - `ProfileStackHeader.index.js` - This is the test file corresponding to this component.
                 - `index.js` - This file will simply export your header file.
                 - `styles` - This is where you'll have styles that are only used in this component. As in React Native we do not have CSS, the folder contains a single `index.js` file which exports the styles.
-                - `components`: If you have abstracted parts of this component into smaller components, use this folder to co-locate them.
+                - `modules`: If you have abstracted parts of this component into React smaller components, use this folder to co-locate them.
```

Comment:
Nitpick, but I would drop the `React` here, as it is implicit.

Reply:
Yep, thanks!

Reply:
Fixed!